client/state/editor/reducer.js

```diff
@@ -75,6 +75,17 @@ export function isIframeLoaded( state = false, action ) {
 	return state;
 }
 
+export function iframePort( state = null, action ) {
+	switch ( action.type ) {
+		case EDITOR_IFRAME_LOADED: {
+			const loaded = action.isIframeLoaded;
```

Comment:
Should we also make sure `action.iframePort` exists too before using it?

Reply:
done with b252597cac